Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.yml

```diff
@@ -0,0 +1,2683 @@
+category: Endpoint
+commonfields:
+  id: Cortex Core - IR
+  version: -1
+configuration:
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: Server URL (copy URL from Core - click ? to see more info.)
+  name: url
+  required: false
+  type: 0
+- display: API Key ID
+  name: apikey_id
+  required: false
+  type: 4
+- display: API Key
+  name: apikey
+  required: false
+  type: 4
+- additionalinfo: The timeout of the HTTP requests sent to Cortex XDR API (in seconds).
+  defaultvalue: '120'
+  display: HTTP Timeout
+  name: timeout
+  required: false
+  type: 0
+- additionalinfo: For Cortex XSOAR version 6.0.0 and above. If selected, for every
```

Comment:
this parameter (Sync Incident Owners) should probably be removed, so no need to doc review it.